transport/src/main/java/io/netty/channel/AbstractChannel.java

```diff
@@ -349,7 +349,7 @@ public String toString() {
                 srcAddr = remoteAddr;
                 dstAddr = localAddr;
```

Comment:
@trustin  @normanmaurer  maybe we still need it due to some user want know if it is client connection or server connection. so we may need to keep the swap logical but just use <=> <:> instead of => and :>.